ext/nnstreamer/nnstreamer_protobuf/meson.build

```diff
@@ -0,0 +1,22 @@
+if protobuf_support_is_available
+  converter_sub_protobuf_sources = [
+    'nnstreamer_protobuf.cc'
+  ]
+
+  nnstreamer_converter_protobuf_sources = [pb_gen_src]
+  foreach s : converter_sub_protobuf_sources
+    nnstreamer_converter_protobuf_sources += join_paths(meson.current_source_dir(), s)
+  endforeach
+
+  protobuf_util_shared = shared_library('nnstreamer_protobuf',
+    nnstreamer_converter_protobuf_sources,
+    dependencies: [nnstreamer_dep, glib_dep, gst_dep, protobuf_dep],
+    install: true,
+    install_dir: lib_install_dir
```

Comment:
Q: why do you define new install path? 
lib_install_dir = join_paths(nnstreamer_prefix, 'lib')  this line makes prefixed (lib) directory.
recommend to use nnstreamer_libdir instead.

Reply:
Right. I failed to install the `libnnstreamer_protobuf.so`, but the path was incorrectly specified in the `nnstreamer.spec` file.
I removed lib_install_dir. Thank you.